### PR TITLE
Document: Scrollbar: Increase affordance

### PR DESCRIPTION
**Pull request by pedropintosilva**
- Make it thicker
- Make it visible by default

Signed-off-by: Pedro Pinto Silva <pedro.silva@collabora.com>
Change-Id: Ibffead10c09eda979ef8d3d2201173aca3553503

### CODE REVIEW

**Comment by pedropintosilva** [comment]
image

File: loleaflet/src/layer/tile/ScrollSection.ts [hunk 1]
```diff
@@ -320,9 +321,9 @@ class ScrollSection {
 
 		var startX = this.size[0] - this.sectionProperties.scrollBarThickness - this.sectionProperties.edgeOffset;
 
-		this.context.fillRect(startX + this.sectionProperties.roundedDpi, scrollProps.startY - this.sectionProperties.roundedDpi, this.sectionProperties.roundedDpi * 4, this.sectionProperties.roundedDpi);
+		this.context.fillRect(startX + this.sectionProperties.roundedDpi, scrollProps.startY - this.sectionProperties.roundedDpi, 0, this.sectionProperties.roundedDpi);
```
Comment (gokaysatir) [inline loleaflet/src/layer/tile/ScrollSection.ts:324]: I think we can delete this line.

File: loleaflet/src/layer/tile/ScrollSection.ts [hunk 2]
```diff
@@ -320,9 +321,9 @@ class ScrollSection {
...
         this.context.fillRect(startX, scrollProps.startY, this.sectionProperties.scrollBarThickness, scrollProps.scrollSize - this.sectionProperties.scrollBarThickness);
-        this.context.fillRect(startX + this.sectionProperties.roundedDpi, scrollProps.startY + scrollProps.scrollSize - this.sectionProperties.scrollBarThickness, this.sectionProperties.roundedDpi * 4, this.sectionProperties.roundedDpi);
+        this.context.fillRect(startX + this.sectionProperties.roundedDpi, scrollProps.startY + scrollProps.scrollSize - this.sectionProperties.scrollBarThickness, 0, this.sectionProperties.roundedDpi);
```
Comment (gokaysatir) [inline loleaflet/src/layer/tile/ScrollSection.ts:326]: If i am not wrong, we can delete this line too.

**Review by gokaysatir — APPROVED** [review]

**Comment by pedropintosilva** [comment]
thanks @gokaysatir  I have changed and I also noticed that somehow setting to `true` `this.sectionProperties.drawVerticalScrollBar` in the `onInitialize()` is not enough to make the scrollbar visible by default. The sidebar appears from on the initialization if I set  `this.sectionProperties.currentAlpha` to `true` but then after some time if disappears  (as it' calling fade out and animating it after some time)

**Comment by pedropintosilva** [comment]
this has been already merge to co-6-4 via https://github.com/CollaboraOnline/online/pull/2726/commits

**Pull request closed without merging**